poetry.lock

```diff
@@ -25,7 +25,7 @@ tests-no-zope = ["cloudpickle", "coverage[toml] (>=5.0.2)", "hypothesis", "mypy
 
```

Comment:
This isn't related to this PR. Make an issue, or ask on github. 

However I don't think running updates in tests makes much sense, especially since it would mean modifying the branches of people's forked repos, forcing the contributors to potentially have to git pull, or worse, be resolving merge conflicts if they also modified poetry.lock. I don't really want to impose this on contributors,  as especially for new contributors, this can be quite challenging. Also, github allows the contributor to disable main repo from making edits to their forked branch, meaning CI like this could even end up failing if people have this disabled.

Reply:
My current solution does not work with PRs, and it should rely on whether is token set, if it is not - fail silently.

Reply:
I don't love the idea of CI commiting to master, not even I, as a maintainer can do that, as it's a protected branch (this would likely block CI from committing there too anyway), meaning it would have to open a PR if there was some actual change made. At that point, we might as well stick with dependabot.